files/etc/systemd/system/sidekiq.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Sidekiq
+After=network.target
+
+[Service]
+Type=simple
+WorkingDirectory=/srv/shop
+EnvironmentFile=/etc/workarea.env
+ExecStart=/srv/shop/bin/bundle exec sidekiq -e production
+ExecReload=/usr/bin/kill -TSTP $MAINPID
```

Comment:
most of this stuff is from [Sidekiq's systemd example service](https://github.com/mperham/sidekiq/blob/master/examples/systemd/sidekiq.service)